bin/strings-pull-project.sh

```diff
@@ -11,7 +11,7 @@ OUTPUT_DIR=src/code/utils/lang
 # tr: Turkish
 # zh-Hans: Simplified Chinese
 # zh-TW: Traditional Chinese (Taiwan)
-LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "tr" "zh-Hans" "zh-TW")
+LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "tr" "zh-Hans" "zh-TW" "th")
```

Comment:
```suggestion
LANGUAGES=("de" "el" "es" "he" "ja" "nb" "nn" "th" "tr" "zh-Hans" "zh-TW")
```
For consistency, keep the list alphabetized and add a comment at line 11 above.

Reply:
I have re-alphabetized the list